Makefile

```
@@ -167,17 +169,25 @@ check-license:
 go/lint:
 	$(GO_ENV) golangci-lint run
```

```suggestion
	golangci-lint run
```
nit: This was an mistake. No need to put GO_ENV there.